tests/modules/create.py

```diff
@@ -65,3 +76,61 @@ def test_modules_create_nfcore_modules_subtool(self):
     assert os.path.exists(
         os.path.join(self.nfcore_modules, "modules", "nf-core", "star", "index", "tests", "main.nf.test")
     )
+
+
+@mock.patch("rich.prompt.Confirm.ask")
+def test_modules_migrate(self, mock_rich_ask):
+    """Create a module with the --migrate-pytest option to convert pytest to nf-test"""
+    pytest_dir = Path(self.nfcore_modules, "tests", "modules", "nf-core", "samtools", "sort")
+
+    # Clone modules repo with pytests
+    shutil.rmtree(self.nfcore_modules)
+    Repo.clone_from(GITLAB_URL, self.nfcore_modules, branch=GITLAB_SUBWORKFLOWS_ORG_PATH_BRANCH)
+    old_main_nf = Path(self.nfcore_modules, "modules", "nf-core", "samtools", "sort", "main.nf")
```

Comment:
maybe easier if we put assign a variable for `Path(self.nfcore_modules, "modules", "nf-core", "samtools", "sort")` and reuse that here?